README.md

```diff
@@ -36,6 +37,14 @@ Here you will find all resources which will be given out throughout your time he
 * Golden Gate Instructors 
     * golden-gate-instructors@appacademy.io
 
+## Cohort Calendar
+**Your Cohort's Calendar is found on the README of your Cohort's Branch**
+1. To get to your branch, go to the top of the page and click the branch dropdown
+2. Then, click on the branch with the name of your Cohort's start date
+    + This process is illustrated below
+ ![calendar](./images/calendar-branch.png)
+3. The Calendar link will then be on the README
```

Comment:
What do you think about adding one more picture here that shows what the calendar link looks like in the README.  Do you think it would be more helpful to add the picture or do you think it's fine with just the third instruction? If you think it would be more helpful you should go ahead and add it but if you think it's fine then I can go ahead and merge it now.  Just let me know.

Reply:
Yeah I was thinking about that but it's the only thing on the README and pretty in your face once you click the link. I can add one if we want tho

Reply:
I guess just to be safe we should add a picture. If you could add one that would be great.  Thank you in advance.

Reply:
will do!